telegram/ext/_jobqueue.py

```diff
@@ -766,6 +769,13 @@ def __init__(
 
         self._job = cast("APSJob", None)  # skipcq: PTC-W0052
 
+    def __repr__(self) -> str:
+        name_infix = f"Job name {self.name}. " if self.name != self.callback.__name__ else ""
+        return (
+            f"Job ID {self.job.id}. {name_infix}Callback name {self.callback.__name__}. "
+            f"APS job trigger: {self.job.trigger}. Next run: {self.next_t}."
```

Comment:
The "next run" information is valid only temporary, i.e. it depends on the time the job is printed (and iirc it.s not even available until the scheduler started). I vote to leave it out.